pandasql/core.py

```diff
@@ -62,6 +62,7 @@ def compute(self):
                 self._compute_sqlite()
             else:
                 self._compute_pandas()
+        self._input_size = self.memory_usage()
```

Comment:
It's a little weird to have both a per-object `memory_usage` function and a `_input_size` variable that stores its result. Maybe any code that needs the memory usage should just call `memory_usage`?

Reply:
I included this because the pd.memory_usage call can be slow, so it seems like it might be worth it to cache the result

Reply:
I'd say remove the `_input_size` for now. We can (either now or later) ensure that the memory usage computation is just done once.